Onboarding + redesign

elzadj opened the pull request:
# Description

How to build starter sites:
(note: main plugin should have the packages installed for it to work)

- Go to maxi-blocks/core/admin/starter-sites folder
- Put the same .env there 
- ```npm install```, wait for it to finish
- ```npm run build```, wait for it to finish

May need npm install --save-dev babel-preset-react-app @babel/plugin-proposal-private-property-in-object

Only tests for now

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Added a detailed System Status Report with environment diagnostics, frontend asset inspection, and report download/copy.
  - Implemented a comprehensive WordPress starter site import system via REST API, including content, styles, images, and patterns.
  - Introduced a new onboarding Quick Start wizard with multi-step guidance and system checks.
  - Enabled theme and WordPress Importer plugin installation/activation from the admin dashboard.
  - Added AI integration settings with OpenAI API key, model selection, and website identity fields.
  - Launched a new Cloud Library interface with search, filtering, preview, and import capabilities for starter sites.

- **Enhancements**
  - Modernized admin dashboard UI with responsive, accessible redesign and new styling.
  - Centralized color variables and CSS for consistent theming across admin and onboarding pages.
  - Improved allowed HTML attributes for buttons and SVG color handling with CSS variables.
  - Expanded settings pages with clearer instructions, grouped options, and font management.
  - Added new React components and UI controls for improved admin experience and onboarding flow.

- **Bug Fixes**
  - Prevented errors when dashboard elements are missing during UI updates.
  - Enhanced content sanitization to handle null or empty values safely.

- **Style**
  - Overhauled admin and onboarding styles with new variables, layouts, and responsive adjustments.
  - Updated block assets to leverage CSS variables for colors and themes.

- **Chores**
  - Refined test configurations and ignore patterns; updated translation strings for new features.
  - Improved code consistency, readability, and maintainability without altering core functionality.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

elzadj committed:
Status report styles

kyrapieterse committed:
Hope its ok

Added some styles

removed really old content that is unneeded under the settings tab

Hope they are ok 🥹

kyrapieterse committed:
Button hovers

hooovers

elzadj committed:
fixes into dashboard

elzadj committed:
Fix Inter font paths

kyrapieterse committed:
Welcome screen hover and removed border on the video image hover

Here she is

kyrapieterse committed:
test

kyrapieterse committed:
foooontttsttstststts

kyrapieterse committed:
Minor input fields update

kyrapieterse committed:
Backing up

kyrapieterse committed:
Variables

kyrapieterse committed:
quick start backup (50% done)

kyrapieterse committed:
Onboarding style ready for Svits 👀👀

kyrapieterse committed:
Added the header with logo and buttons to the quick start

kyrapieterse committed:
Padding 22px 22px 22px 32px;

kyrapieterse committed:
Tweaklings

elzadj committed:
Fix steps after renaming

kyrapieterse committed:
Start of responsive - backup

elzadj committed:
Fix Remove logo and icon buttons

elzadj committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

kyrapieterse commented:
![image](https://github.com/user-attachments/assets/66d4cef0-9683-450c-ac2e-00a47accff3e)

kyrapieterse committed:
Quick start + responsive

Best to check it please 🙏🙄🙄

kyrapieterse committed:
Backup - WIP - not ready yet

kyrapieterse committed:
Revert "Backup - WIP - not ready yet"

This reverts commit 5d2697461dfd540a93995ddd3fb6a7082d2c0da1.

kyrapieterse committed:
Maxi AI and Settings Accordion/background and main content area responsive

kyrapieterse commented:
I cant for the life of me figure out why the Insert Google Maps API Key here input is not taking the responsive settings - please check and help!!! 

https://media.tenor.com/Ucrf6GJT9dQAAAP1/mochi-mochimons.mp4

kyrapieterse committed:
Minor update to the responsive inputs

kyrapieterse committed:
remove test bg colour

kyrapieterse committed:
more responsive tweaks - better responsive nav

kyrapieterse committed:
Better nav/header with clamp

kyrapieterse committed:
More responsive with clamp

kyrapieterse committed:
dont know this is a just in case backup lol

kyrapieterse committed:
var in the styes for status report

kyrapieterse committed:
System status 💪

kyrapieterse committed:
First toe into the main editor new UI revamp

Changed the orange colour that was a hex to variable is the biggest change here

kyrapieterse committed:
indicators upgrade

kyrapieterse committed:
Second backup of the editor

Radically renamed and fixed the var names. Updated some of the files. Still quiet a few to go so this is a backup pushy

kyrapieterse committed:
Tesssssssssssssssssttttttttttttttttttttttttssssssssssssssssssssss

kyrapieterse committed:
backup

kyrapieterse committed:
backup. most on the settings panel updated with new variables and some minor fixes

kyrapieterse committed:
Friday night backup

kyrapieterse committed:
90% of background layer upated. still busy this is just a backup

kyrapieterse committed:
small changes and a backup

kyrapieterse committed:
fixed the missing text border

kyrapieterse committed:
SVG from G fill='var(--maxi-grey-4-color)'to Maxi var black

kyrapieterse committed:
Alignment control ui

kyrapieterse committed:
advanced colour and tooltip

kyrapieterse committed:
quick back up before a find and replace 🥹

kyrapieterse committed:
anther quick back up

kyrapieterse committed:
Up to C of components

kyrapieterse commented:
![image](https://github.com/user-attachments/assets/c9522f9c-5808-4ae3-b9c1-c3ce05e1e25c)
I cant for the life of me find where this is coming from @elzadj 🙄🙄🙄🙄

kyrapieterse committed:
minor tweakies

kyrapieterse committed:
more colour updates to new variables

kyrapieterse committed:
repeater colours

kyrapieterse commented:
I updated the row title and I somehow broke the sidebar 🫠🙄 please help me @elzadj 🙏

![image](https://github.com/user-attachments/assets/88c15fd5-7ffa-4a11-8c2a-46b527841464)

elzadj committed:
Fix Column picker

elzadj committed:
Fix advanced numebr control styles

elzadj committed:
Fix maxi-block border-radius on hover

elzadj committed:
Fix sc colour presets test

kyrapieterse committed:
small colour var updates

kyrapieterse committed:
Morning backup - small colour/var updates

kyrapieterse committed:
more colours

kyrapieterse committed:
pushy

kyrapieterse committed:
toolbar part 1

kyrapieterse committed:
new toolbar icons and other same fixes

kyrapieterse committed:
starting the cloud UI refresher. Yeah baby yeah

kyrapieterse committed:
cloud backup

kyrapieterse committed:
backup before I start. Friday lets f**king gooooooooooo

kyrapieterse committed:
backup

kyrapieterse committed:
cloud backuparou

kyrapieterse committed:
more colour variables  🤪

kyrapieterse committed:
fix the missing dashed line

kyrapieterse commented:
A list of thing I would like to change maybe later but currently low priority 

.ais-SearchBox-reset

kyrapieterse committed:
working on the icon clud now - pre start backup

kyrapieterse committed:
Revert "working on the icon clud now - pre start backup"

This reverts commit 01257b1f5c3fee37d563a576490f93f13c6bfd3b.

kyrapieterse committed:
Reapply "working on the icon clud now - pre start backup"

This reverts commit c42c1419bc01f7d3dfce9a77b81768bd40e87b8e.

kyrapieterse committed:
backup 12.16

kyrapieterse committed:
trying to kill the green

kyrapieterse commented:
![image](https://github.com/user-attachments/assets/5626ed68-d0ff-435d-91d4-32029a8b63ad)

kyrapieterse committed:
clouds

kyrapieterse committed:
more cloud fixes including style cards

kyrapieterse committed:
small edits

elzadj commented:
Cloud Block: 
Need to do something with the logo, it's unreadable (in the blocks inserter too), and the button padding:
![Edit-Page-‹-MaxiBlocks-local-site-—-WordPress-03-03-2025_11_40_AM](https://github.com/user-attachments/assets/5896a4a6-9b0c-45f9-9a67-ffa45f0c1ec8)

The cloud modal (and others too):
![Edit-Page-‹-MaxiBlocks-local-site-—-WordPress-03-03-2025_11_40_AM (1)](https://github.com/user-attachments/assets/95619d78-4c16-410c-a523-83008d839dc7)

The patters preview is broken:
![Edit-Page-‹-MaxiBlocks-local-site-—-WordPress-03-03-2025_11_43_AM (1)](https://github.com/user-attachments/assets/cf8e6429-845e-493b-9cee-f8149ee91a1b)

elzadj committed:
Fix Settings width in dashboard

elzadj committed:
Fix pattern preview iframe styles

elzadj committed:
Update the placeholder test snap

kyrapieterse committed:
remove SC title

kyrapieterse commented:
I have 🤔
![image](https://github.com/user-attachments/assets/e2931ad0-9d47-4c17-8cc0-e6de9c1beaf4)

kyrapieterse committed:
toolbar part one

kyrapieterse committed:
more toolbar

kyrapieterse committed:
icooons

kyrapieterse committed:
toolbar part2

kyrapieterse committed:
More UI fixxes

kyrapieterse committed:
last one for tonight

kyrapieterse committed:
mid morning backup

kyrapieterse commented:
![image](https://github.com/user-attachments/assets/54502ab4-eed7-4791-84a5-8d897f1da5af)
ib?

kyrapieterse committed:
more UI style updates

kyrapieterse committed:
morning update

kyrapieterse committed:
colour backup

kyrapieterse commented:
> need to fix missing --maxi-whisper-green-alt colour

done ✅

kyrapieterse committed:
pushy push push

kyrapieterse committed:
Map refresh 🗺️

https://media.tenor.com/274XvFrQJV8AAAP1/kermit-map.mp4

kyrapieterse committed:
Add new interaction button upgrade

kyrapieterse committed:
ui

kyrapieterse committed:
small UI update

kyrapieterse committed:
accordion round one

kyrapieterse committed:
UI small bits and bobs

kyrapieterse committed:
quick backup before find and replace

elzadj committed:
Fix svg icon bg test

elzadj committed:
Fix icon colour toolbar circles

kyrapieterse committed:
min accordion backup and search ui

kyrapieterse commented:
still needs help.

![image](https://github.com/user-attachments/assets/11c01f35-dfbb-4249-a11f-4eb515b87745)
![image](https://github.com/user-attachments/assets/846c9c5f-d0da-4584-b382-b967842b1089)

kyrapieterse commented:
Things I need help with

- [ ] Style card editor weird scroll https://jam.dev/c/545520c1-316b-4494-820b-c0f749502c37
- [ ] I dont get the dot or the layer bug https://jam.dev/c/91fb4daa-10c7-4cac-9ece-702a1f2c93d8
- [ ] The accordion (I aged trying to crack this one!!) https://jam.dev/c/ce32fe11-7ead-4460-884a-29e102a17698 basicaly the paddings/margins are off on most them :(

kyrapieterse committed:
Style cards refresh

elzadj committed:
Merge master and fix conflicts

elzadj committed:
Fix normal / hover / acrive tabs in the inspector sidebar

kyrapieterse committed:
PHP spaces

elzadj committed:
Fix accordion settings styles

kyrapieterse committed:
Merge branch 'onboarding-redesign' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

elzadj committed:
starters site dev mode

elzadj committed:
Merge branch 'onboarding-redesign' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

elzadj committed:
Fix maxi block test

kyrapieterse committed:
starters backup

kyrapieterse committed:
Merge branch 'onboarding-redesign' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

kyrapieterse committed:
starters

elzadj committed:
resolve conflicts with master

elzadj committed:
resolve conflicts with master

elzadj committed:
Fix tests (maybe)

elzadj committed:
Fix button sidebar settings styles

elzadj committed:
Fix IB sidebar settings styles

elzadj committed:
Fix container, row, link settings styles

elzadj committed:
revert icons default colour to orange

elzadj committed:
Fix video settings styles

elzadj committed:
Fix maxi ai dashboard settings styles

elzadj committed:
Fix starters styles, update tests

elzadj commented:
Ready for tests. Need to check:

- The whole plugin UI in the Editor (and FSE)
- New MaxiBlocks Dashboard
- Onboarding process
- Starter sites import 

How to build starter sites:

- Go to maxi-blocks/core/admin/starter-sites folder
- Put the same .env there
- ```npm install```, wait for it to finish
- ```npm run build```, wait for it to finish
May need ```npm install --save-dev babel-preset-react-app @babel/plugin-proposal-private-property-in-object```

Don't forget to clean the cache before tests to re-load styles.

Left to do:
- Maxi Dashboard Welcome page
- New images previews for some blocks
- System report php version 
- probably something else too...

Olekrut reviewed (changes requested):
- [x] 1) FSE. Second horizontal scroll
 ![image](https://github.com/user-attachments/assets/c0a1694c-cd4d-4cbb-8961-00d15a2b74c1)
- [x] 2) reset colour button is too close to colours
 ![image](https://github.com/user-attachments/assets/86b3e16e-e5d3-4cfc-b2c3-98238f49d6bf)
- [x] 3) Is need to update preview image for text block?
 ![image](https://github.com/user-attachments/assets/b05d1774-291d-493f-aa04-b5b0159682da)
- [x] 4) As for me grey (2) is not very contrasting, hard to read sometimes, maybe better to make black everywhere (1) ?
 ![image](https://github.com/user-attachments/assets/02c82f25-61ff-4516-888c-b07f9d8b580c)
- [x] 5) Tabs SC and Export have bad UX as for me, if to open both need 2 clicks, that will help to see another tab. 
 https://jam.dev/c/428fe293-55a2-4de6-b57f-17ede4e94d35
- [x] 6) Weird active and hover styles, Toolbar edit 
 https://jam.dev/c/56aaef3c-e57d-4699-9a49-77b2a73f41b9
 ![image](https://github.com/user-attachments/assets/e02eddb1-3412-4f95-9c0b-775d7f4cc37a)
- [x] 7) Maxi AI writer - range control higher than dropdown 
https://jam.dev/c/413aecca-6059-47f3-876b-53360e2ddcf8
 ![image](https://github.com/user-attachments/assets/e2d1123f-0ce6-4c73-9643-915cce39bcea)
- [x] 8) Delete Icon is not align
 ![image](https://github.com/user-attachments/assets/4ed39406-a4eb-4aa2-80e4-02ed05293479)
- [x] 9) Some of the tabs, buttons and input fields don't have transition ( is not very important )
 https://jam.dev/c/e78b6392-801b-46e0-9913-d0e0fd3403bc
- [x] 10) Input fields have weird border on hover ( corners )
 ![image](https://github.com/user-attachments/assets/21b141d3-dcd3-4b7f-9288-0d86579a508d)
- [ ] 11)
- [x] 11.1) Scroll effects tab, guess, need to make bigger font-size as earlier, as everywhere.
 ![image](https://github.com/user-attachments/assets/b2cd4940-7b41-47db-8c7a-88231e52aaef)
- [x] 11.2) And to add hover for delete button
 ![image](https://github.com/user-attachments/assets/99313ff2-34e8-4fe7-a211-05af6c4081c6)
- [x] 12) Clip-path styles looks weird, especially points
 https://jam.dev/c/735f954e-cec3-486c-b33b-1c0d7e80f010
- [x] 13) Take a look on outline for text block - it is appearing only if to add some text
 https://jam.dev/c/fd86d529-67c0-4bad-a3d5-015c622abfc8
- [ ] 14) And not sure, but feels like speed is a bit slower - everything is not smooth, it's more jerky
 Tested on  Concert CCTP-PRO-01
 Master - 32sec
Branch - 43sec
- [x] 15) Maybe need to make for this button similar styles 
 ![image](https://github.com/user-attachments/assets/ff6fb55e-38c9-45d5-94b3-bc6a1e7f7ef1)
- [x] 16) Is it only for Heading and IB?
 ![image](https://github.com/user-attachments/assets/ac128181-2124-488b-b82a-c8d7af383128)
- [x] 17) Transform Scale
Looks like it is not square ( 
https://jam.dev/c/0d22d2ce-27f4-4293-85e9-67214ad4e49c
- [x] 18) Transform Translate
 I think you need to make the moving element more contrasting and maybe add hover
https://jam.dev/c/f2759cff-f95f-45c9-abc7-d1e5bc2c6f5d
- [ ] 19) Scrolling of link's list by wheel. If cursor inside link`s list area > page should not be scrolling 
 https://jam.dev/c/146ae243-9143-4d13-96e8-35922106fb71
- [x] 20) Need to make less opacity or smth for hidden layer
![image](https://github.com/user-attachments/assets/c2211577-1633-40f8-bc8d-2f06596a7b3e)
- [x] 21) What is it? Image or video?
![image](https://github.com/user-attachments/assets/ee3ace46-553b-492e-8569-bdba02cfd07d)
- [x] 22) UI
 ![image](https://github.com/user-attachments/assets/177e1d32-c307-428c-a5e0-de5805bb6c5d)
- [ ] 23)

kyrapieterse commented:
@Olekrut we have a separate issue for the preview images already

kyrapieterse committed:
2) reset colour button is too close to colours

kyrapieterse committed:
4) As for me grey (2) is not very contrasting, hard to read sometimes, maybe better to make black everywhere (1) ?

kyrapieterse committed:
6) Weird active and hover styles, Toolbar edit

kyrapieterse committed:
11.1) Scroll effects tab, guess, need to make bigger font-size as earlier, as everywhere.

kyrapieterse committed:
11.2) And to add hover for delete button

kyrapieterse committed:
12) Clip-path styles looks weird, especially points

My bad. Should be fixed now

kyrapieterse committed:
18) Transform Translate I think you need to make the moving element more contrasting and maybe add hover

kyrapieterse committed:
New colour I am testing - Feedback welcome

elzadj committed:
Merge master and restore admin colour variables

kyrapieterse committed:
new green take 2

kyrapieterse committed:
admin colours

elzadj committed:
Fix indicators

elzadj committed:
More css fixes

kyrapieterse committed:
3) Is need to update preview image for text block?

All updated

elzadj committed:
Merge branch 'onboarding-redesign' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

kyrapieterse committed:
tiny colour update

elzadj committed:
Advanced css and transform controls

elzadj committed:
Merge branch 'onboarding-redesign' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

kyrapieterse committed:
link

kyrapieterse commented:
![image](https://github.com/user-attachments/assets/cf05a0a6-92e9-4113-b5e8-be56cbb9af0e)
 tell me where it is so I can learn

elzadj committed:
Merge branch 'onboarding-redesign' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

elzadj committed:
revert link control styles

elzadj committed:
Fix fse double horizontal scroll

elzadj commented:
point 5 - moved the export window, also hid the Export button, since it's for editors only, you can show it with css.
point 9 - fixed some, but probably not all, a list would be nice.
point 19 - can't change it, it's a WP component.

elzadj committed:
Fix hover for toolbar link input

elzadj committed:
Fix broken lh for quick start

kyrapieterse committed:
colour

kyrapieterse committed:
Icon position

kyrapieterse committed:
small one

elzadj committed:
Fix conflicts with master

elzadj committed:
Remove broken style from quick start

elzadj committed:
remove ;

Olekrut reviewed (changes requested):
- [x] №13) is not fixed
https://jam.dev/c/0d9ab6d3-13db-47b3-8d2d-b1355ef8d38e

- [x] 23) tick in cloud looks small, 
![image](https://github.com/user-attachments/assets/592e4e87-8522-48c3-a78a-c2c1d7055549)
i guess, 20px looks better
![image](https://github.com/user-attachments/assets/7f6004eb-8330-4aee-b395-a830d97cf662)
- [x] 24) Active styles for tab is border(1) is not very noticeable as for me. Hover styles looks ok(2)
![image](https://github.com/user-attachments/assets/3aedfe37-3d74-46e9-b2cc-131730efd553)
- [x] 25) Hover for + on between block appender
https://jam.dev/c/afac30b1-f3f9-4a33-9f80-20ec1f6fda29
- [x] 26) Almost for all dropdown and input fields > there are disappeared top border-line
![image](https://github.com/user-attachments/assets/686eaae6-3926-45a8-8fa1-c3077e08ea99)
- [x] 27) Maxi AI writer - Generate button on hover > need another colour for text
![image](https://github.com/user-attachments/assets/bde4d1a1-3bae-48c8-909f-2fa3e898d029)
- [x] 28) Tabs where is bottom line on hover > it is dragging > line should be in absolute or need to change hover and active state for them
https://jam.dev/c/e19a6afb-bfd6-43c3-b3fa-d53326b3252e
![image](https://github.com/user-attachments/assets/15f72e3a-0d29-4461-a065-1f92f709ea50)
29) 
- [x] 29,1) For all dropdown lists > would be good to add changing text on hover ( I guess, black )
![image](https://github.com/user-attachments/assets/e411759f-39f4-47d6-8868-fc8b7bb45eca)
- [x] 29,2) Some of the dropdown lists have native WP styles. They are good, but should not be similar to new styles?
![image](https://github.com/user-attachments/assets/0211b961-9334-4f37-b466-736817f770da)
- [x] 30) Almost for all reset buttons, we have just changing colour of arrow
![image](https://github.com/user-attachments/assets/c5c1e98c-a51a-44a9-a981-3a91a177686c)
But for some of them we have also changing bg colour of button
![image](https://github.com/user-attachments/assets/56df6cbb-7324-40df-89ce-919e1874f15c)
Should not be the same styles everywhere?
- [x] 31) Scroll effects > units dropdown > need to add text-transform: uppercase; as everywhere
![image](https://github.com/user-attachments/assets/a0de5f66-79e3-440c-937f-37b19865d419)
- [x] 32) I guess, would be better to make hover styles as on our site,
![image](https://github.com/user-attachments/assets/a68e8157-d01d-47d4-a40d-e9ae3ec5f2f0)
- [ ] 33) Why only for Quick start we have drop down in admin panel?
![image](https://github.com/user-attachments/assets/ba9852a2-5e94-4674-a63d-668e76d1d00a)
- [x] 34) Why we don't have nav for Quick start?
![image](https://github.com/user-attachments/assets/fd41500d-7d5d-47d9-8f6f-7a9990a27690)
- [x] 35) Where are hovers for these buttons?
![image](https://github.com/user-attachments/assets/a2cedb06-a64e-4cfd-81b4-0e7f611d31f8)
- [x] 36) Elements without transition ( continue for point 9, some of the points can be for the same elements, but better I will add them just in case ), list: 
- [x] 36.1) 
![image](https://github.com/user-attachments/assets/84d443f8-f658-4e9f-a7f8-6f1313bb900e)
- [x] 36.2) Would be good to add some for accordions
https://jam.dev/c/cd660540-cc9c-41e2-ae6b-3eb0f3a0d59d
- [x] 36.3) 
![image](https://github.com/user-attachments/assets/dbdaf08e-5725-4c87-85e0-f179b1caf775)
- [x] 36.4) hover-out transition
https://jam.dev/c/88d3da09-62e8-404f-9c5b-01eff09bc9b3
- [x] 36.5) 
![image](https://github.com/user-attachments/assets/637d8c61-ce00-42a3-98fc-57ae971e13a2)
- [x] 36.6) transition and i don't like hover for remove button, maybe to change bg as well?
![image](https://github.com/user-attachments/assets/da809a9f-70ec-41ec-b3cc-68a92887028a)
- [x] 36.7) Alignment
![image](https://github.com/user-attachments/assets/b06ee112-f423-42fd-9412-06b0a746ef42)
- [x] 36.8) Dropdown fields ( EVERYWHERE!!! )
![image](https://github.com/user-attachments/assets/51adefff-6312-4ad1-951e-a32161ec7fbe)
- [x] 36.9) 
![image](https://github.com/user-attachments/assets/fe9bed56-d3a8-4c0e-b9ff-794c7389b12c)
- [x] 36.10) sidebar tabs
![image](https://github.com/user-attachments/assets/e7874c9a-35a8-4a7d-96df-6a8b8fd7d53c)
- [x] 36.11) 
![image](https://github.com/user-attachments/assets/10a40ef9-e451-4333-8c09-aecc3e4f2f99)
- [x] 36.12) 
![image](https://github.com/user-attachments/assets/fc4a1171-be87-463b-92ec-f8e52fb1111c)
- [x] 36.13) 
![image](https://github.com/user-attachments/assets/e6244a53-c1e8-4c3d-a874-b138d57af737)
- [x] 36.14)
- [x] 36.14.1)
![image](https://github.com/user-attachments/assets/c6bf84e4-93e8-4741-ad3f-f8b73c287517)
- [x] 36.14.2)
![image](https://github.com/user-attachments/assets/80ec1932-ffc1-4dcd-b43f-3b653aaa173f)
- [x] 36.15) 
![image](https://github.com/user-attachments/assets/0446f284-3e00-4282-8d16-49dd0fd10680)
- [x] 36.16) 
![image](https://github.com/user-attachments/assets/13baeec0-d40e-430c-9a68-3b784dbfc0cb)
- [x] 36.17) 
![image](https://github.com/user-attachments/assets/285477c8-0bbe-4064-91b6-59d2ca252a6e)
- [x] 36.18) 
![image](https://github.com/user-attachments/assets/1fd4b60c-593f-4b32-a635-ad71f032beed)
- [x] 36.19) 
![image](https://github.com/user-attachments/assets/d516b8e7-c1d4-4273-826f-6bf04ff268ee)
- [x] 36.20) 
![image](https://github.com/user-attachments/assets/3871160b-3271-414c-a97d-342409d5339d)
- [x] 36.21) 
![image](https://github.com/user-attachments/assets/458d2bf4-86c9-4ccd-8b26-6811ec2d3b23)
- [x] 36.22) 
![image](https://github.com/user-attachments/assets/d464aca8-4a52-4e3e-9409-dda2b98fb2a0)
- [x] 36.23) 
![image](https://github.com/user-attachments/assets/acd35f05-dee2-4b70-a26d-ce37f9d7cfa5)
- [x] 36.24) Icon and Image
![image](https://github.com/user-attachments/assets/3fb00293-8db2-42d0-8f75-86d544367c32)
- [x] 36.25) 
![image](https://github.com/user-attachments/assets/95965b68-3c79-4c78-a9e6-976707e7dbf4)
- [x] 36.26) 
![image](https://github.com/user-attachments/assets/b0822084-5c77-424f-aa66-ab3e6979662b)
- [x] 36.27) 
![image](https://github.com/user-attachments/assets/caf5c64a-d269-45af-8a3f-07cb977f5c60)
- [x] 36.28) 
https://jam.dev/c/42747370-ea23-4bb6-94a1-42306b087b1a
- [ ] 36.29) Toolbar > alignment > weird behaviour, that depend on which is active
https://jam.dev/c/d1519125-b7a8-480d-8216-90a48ac4d48e
- [x] 36.30) 
- [x] 36.30.1) hover for icons(1) 
![image](https://github.com/user-attachments/assets/b59ead72-24c6-4cb9-8813-ae8ae86f5408)
- [x] 36.30.2) need to add active icon colour for gradient icon(2)
![image](https://github.com/user-attachments/assets/b02b2104-c45f-4e48-81e3-8f6fbf2547cf)
- [x] 36.31) Callout arrow does not have transition
![image](https://github.com/user-attachments/assets/bd260ccd-eb20-4bd4-a68a-7fadc3475ff8)
- [x] 36.32) 
![image](https://github.com/user-attachments/assets/f3ed35eb-9a14-4d17-98cf-7325720eb9f7)
- [x] 36.33) 
![image](https://github.com/user-attachments/assets/672025f7-1343-40f0-9346-6d53dacce987)
- [x] 36.34) 
![image](https://github.com/user-attachments/assets/32910d94-e0aa-418f-ab05-83eb9fc91b8e)
- [x] 36.35) 
![image](https://github.com/user-attachments/assets/4606edf0-2a5a-41a8-98c3-b2ab69c1aa2a)
- [x] 37) Alignment
![image](https://github.com/user-attachments/assets/d2f9f7f2-b4c6-435f-84c6-ba3ccaec44de)
- [x] 38) IDK, maybe need to add some links or info how to update them? Predict that we will get some tickets about it, if we are showing that, because it is Warning LOL
![image](https://github.com/user-attachments/assets/dac9a11f-feb9-49ee-bf94-d499cde361b7)
- [x] 39) Maybe need to make it wider for big screens? 
![image](https://github.com/user-attachments/assets/11e66294-89bd-4f12-8685-cc46d7e97f2c)
- [x] 40) Need to add hover for link
- [x] 40.1)
![image](https://github.com/user-attachments/assets/2ee46997-e5e1-41de-b9cd-475b0553a19a)
- [x] 40.2) 
![image](https://github.com/user-attachments/assets/a5afdf90-2455-4695-a27b-8309a060dc06)
- [x] 40.3)
![image](https://github.com/user-attachments/assets/ea415f1f-be77-4841-bfa3-d96badd9bba6)
- [x] 41) Maybe need to add some space?
![image](https://github.com/user-attachments/assets/1a641ce2-7246-4c71-872a-02b8fdf1031e)
- [x] 42) I guess, need to make another colour for hover of nav or change regular colour to black
![image](https://github.com/user-attachments/assets/b6dc1533-bce9-4487-b33d-f483685f1b8c)
- [x] 43) Remove button should be higher than border(hover of icon field) and need to add hover for remove button
![image](https://github.com/user-attachments/assets/1eef85f5-bb93-485e-9928-4469c79e38f6)
- [x] 44) 
- [x] 44.1) I guess, need more contrast colour ( black?)
https://jam.dev/c/88179d55-b48e-4a88-a991-c28e7a2279d6
![image](https://github.com/user-attachments/assets/6b703a8e-4eb4-44c6-bd0d-a282d8952df5)
- [x] 44.2) And is not weird inactive state for stroke icon?
![image](https://github.com/user-attachments/assets/e0668fa4-d3d8-4d65-8e76-e88b414a69c0)
- [x] 45) 
- [x] 45.1) Could we make the same styles for these circles in gradient window?
![image](https://github.com/user-attachments/assets/0b03c3cc-abec-4eb6-8fee-10d3f3137a8f)
- [x] 45.2) The same for Set Custom Colour
![image](https://github.com/user-attachments/assets/144393cf-8395-4a0e-8385-79c62ef4df69)
- [x] 46) 
- [x] 46.1) need to add more space
- [x] 46.2) need to make the same styles for hover as everywhere
![image](https://github.com/user-attachments/assets/04977d55-bf07-4fde-8a68-2a2c4521f81f)
- [x] 47) Would be good to add hover for X, because only this has another option in this dropdown field ( the same in toolbar )
![image](https://github.com/user-attachments/assets/2b291d4c-5837-4bda-9781-31163729000e)
- [ ] 48) When we are changing Heading > indicator is appearing for Typography as well 
But as I understand, indicator should appear only if we are changing default styles > typography has default styles for current Heading > so indicator should not appear
![image](https://github.com/user-attachments/assets/228fc472-8e91-4379-8160-82a012d38594)
- [ ] 49) Swapping between H-tags in toolbar is much longer than in sidebar ( maybe the same on master, but hope you will fix it here as well or need to create new issue )
https://jam.dev/c/4fa4a33f-6709-4736-9c20-93e12e5ca8d6
- [x] 50) Need to add hover and active styles
![image](https://github.com/user-attachments/assets/5a7c71fc-9f3c-4945-a270-8dcb85ab959d)
- [x] 51) Need to add styles for small columns
![image](https://github.com/user-attachments/assets/ada043ed-0af4-40a8-8039-4d5db9d48f25)
- [x] 52) Need to add hover
![image](https://github.com/user-attachments/assets/c6e07d04-1f38-4da4-89a0-cce13c85b31d)
- [x] 53) Need to add hover
![image](https://github.com/user-attachments/assets/34af96de-79a3-49e2-a67a-6b19565c0a48)
- [x] 54) In this case hover should work only on dropdown menu
https://jam.dev/c/97025d0e-0d58-46c6-9519-953f43bdd036
- [x] 55)  Would be good to add some hover for bg layer tab
![image](https://github.com/user-attachments/assets/d46fd13d-38a7-4677-8283-a8a57f343839)
- [x] 56) I guess, we should not have hover for elements which are not active at current moment
![image](https://github.com/user-attachments/assets/c2693e14-6bfa-49aa-b6d6-969cd67e77d0)
- [ ] 57) Check indicators for column picker, I guess, they are not working correct, especially for BP tabs
Example, I am creating Row (3 columns) > then I am changing it to 4 columns ( less or more dont matter ) > why is appearing indicator? they are also default. I guess, Indicator should appear only if we are changing - row-gap, column-gap, flex-wrap.
- [ ] 58) Check all indicators > indicators shold appear if user are changing default settings > but we can see that some indicators are appearing, in the moment, after adding block ( Example:  row - column picker, height/width; column - column width, etc ) 
https://jam.dev/c/df27b056-2e55-4953-8cd4-7a38cb5443d3
- [x] 59) I see some weird hover on dropdown menus ( test is dragging )
https://jam.dev/c/42747370-ea23-4bb6-94a1-42306b087b1a
- [x] 60) Hover for angle looks weird
![image](https://github.com/user-attachments/assets/fc927450-045c-48cf-ad52-b37451819fd4)
- [x] 61) points for clip-path are cuting. Is possible to do smth that we would have ability to see them fully and easier to grab them? 
![image](https://github.com/user-attachments/assets/5ff3e3f8-7efb-4754-a3cc-083f99bc0ff9)
- [x] 62) Need to add hover and transition for list settings in loolbar
![image](https://github.com/user-attachments/assets/b5b50218-c4cf-4232-8955-0c37da46edee)
- [x] 63) Maybe need to make the same font-styles in sidebar and toolbar (font-weight?) ?
![image](https://github.com/user-attachments/assets/0d8b4cd2-c8d7-4666-b887-287885d9a33a)
- [x] 64) Another style for reset button
![image](https://github.com/user-attachments/assets/9df55957-8be0-4784-96d9-f87bd3c25138)
- [x] 65) transition for icons and maybe no need to make hover colour for stroke the same as fill?
![image](https://github.com/user-attachments/assets/be18b6ee-e8e5-438c-bf95-5099fa0fc7b9)
- [x] 66) Looks like only this branch
Try open any Custom CSS and type for example: """color:red !important;""" 
cursor has wrong position in the end. Idk, Like text longer than should be or vice versa.
https://jam.dev/c/f93b33c9-0fb3-4269-a745-9f6982364ffe
- [x] 67) Need to add hovers
![image](https://github.com/user-attachments/assets/70a83602-022e-4453-981a-b70caed16e94)
- [x] 68) Alignment
![image](https://github.com/user-attachments/assets/3510d640-acfd-4d1d-81a9-551977a5d100)
- [x] 69) I guess, would be better to make black colour for text of custom link
![image](https://github.com/user-attachments/assets/1742df0c-4c56-4456-b7a7-e0a189cbd0a2)
- [x] 70) Is not better to make this dropdown wider?
![image](https://github.com/user-attachments/assets/6467d4ad-0fa9-42ce-a271-029bb75a3eb0)

kyrapieterse committed:
23) tick in cloud looks small,

kyrapieterse committed:
24) Active styles for tab is border(1) is not very noticeable as for me. Hover styles looks ok(2)

kyrapieterse committed:
25) Hover for + on between block appender

kyrapieterse commented:
@elzadj No idea why my css is making this centered take a look please 
![image](https://github.com/user-attachments/assets/7692add2-11ba-421e-ad95-647c7d4f65ac)

maxi-blocks\src\components\icon-control\editor.scss

kyrapieterse committed:
28) Tabs where is bottom line on hover > it is dragging > line should be in absolute or need to change hover and active state for them

kyrapieterse committed:
30) Almost for all reset buttons, we have just changing colour of arrow

kyrapieterse committed:
31) Scroll effects > units dropdown > need to add text-transform: uppercase; as everywhere

kyrapieterse committed:
32) I guess, would be better to make hover styles as on our site,

kyrapieterse committed:
35) Where are hovers for these buttons?

kyrapieterse committed:
dash hovers

kyrapieterse committed:
Alignment and button hoves

kyrapieterse committed:
36.8) Dropdown fields ( EVERYWHERE!!! )

kyrapieterse committed:
36.10) sidebar tabs

kyrapieterse committed:
small trantions

kyrapieterse committed:
36.15)

kyrapieterse committed:
36.16)

kyrapieterse committed:
36.17)

kyrapieterse committed:
36.19)

kyrapieterse committed:
small transition up to 24

kyrapieterse committed:
36.25)

kyrapieterse committed:
36.26)

kyrapieterse committed:
36.27)

kyrapieterse commented:
@elzadj @Olekrut some notes

38) IDK, maybe need to add some links or info how to update them? Predict that we will get some tickets about it, if we are showing that, because it is Warning LOL

**Let's wait for support LOL**

39) Maybe need to make it wider for big screens? - **it's fine as is**

51) Need to add styles for small columns - **new issue**

 56) I guess, we should not have hover for elements which are not active in current moment - **fine for now as its greyed out**
 
 We already have issues for indicators or this is something new?

kyrapieterse committed:
52) Need to add hover and other small ones

kyrapieterse committed:
55) Would be good to add some hover for bg layer tab

kyrapieterse committed:
button fix

kyrapieterse committed:
small fix

kyrapieterse committed:
Help push

kyrapieterse committed:
clounds

elzadj committed:
Styles for empty text

elzadj committed:
Fix prompt button colour on hover

elzadj committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

elzadj committed:
Fix hover for react select focused option

elzadj committed:
Gradient remove points styles

elzadj committed:
Background condtol

kyrapieterse committed:
36.31) Callout arrow does not have transition

kyrapieterse committed:
36.32)

kyrapieterse committed:
36.35)

kyrapieterse committed:
fix blue text in hover of responsive hover

kyrapieterse committed:
63) Maybe need to make the same font-styles in sidebar and toolbar (font-weight?) ?

elzadj commented:
What's "weird" here? @Olekrut 

> 36.29) Toolbar > alignment > weird behaviour, that depend on which is active
> https://jam.dev/c/d1519125-b7a8-480d-8216-90a48ac4d48e

elzadj committed:
36.30.2) need to add active icon colour for gradient icon

elzadj committed:
37) Alignment

elzadj committed:
41) add some space to local fonts size

elzadj committed:
43) Remove button should be higher than border(hover of icon field) and need to add hover for remove button

elzadj committed:
44) darker icons and active state fix

elzadj committed:
47) hover for x in the font selector

elzadj commented:
48. The default attribute is "p", when you change it, it shows the indicator, because it's not the default "p" any more.
49. New issue, please.

elzadj committed:
46) gradient control

elzadj committed:
Add hover for auth email input

elzadj committed:
52, 54 link control

Olekrut commented:
- [ ] 71) 
- [ ] 71.1) need to add hovers
- [ ] 71.2) this window has cursor:point everywhere, not only toggles and button
![image](https://github.com/user-attachments/assets/89624518-54de-498e-8a8c-e1fc26eabd65)
- [ ] 72) Ways for these 2 links are not correct
![image](https://github.com/user-attachments/assets/0596b974-d0c3-45f2-8348-343234eb953b)
- [ ] 73)
- [ ] 73.1)  Need to add transition for buttons and change hover for ?-icon and add cursor: point for "Sign out" button
![image](https://github.com/user-attachments/assets/b731a11b-8f51-4820-a140-1427b67830ad)
- [ ] 73.2) pop-up, transition for X
![image](https://github.com/user-attachments/assets/5d56016c-bf7f-49eb-90c1-9478f5b43c79)
- [ ] 74) 
- [ ] 74.1) Need to add transition for X
- [ ] 74.2) Are there 2 close icon ( X ) ?
![image](https://github.com/user-attachments/assets/99efc57b-3078-464a-b4d0-1e3bb4f81e99)
- [ ] 75) Need to add transition for icons transform
![image](https://github.com/user-attachments/assets/fee2cf40-ba95-4446-ab60-2987a5734d5f)

elzadj committed:
59, 60 hover

elzadj committed:
Fix clip-path points

elzadj committed:
hover and transition for list settings in toolbar

elzadj committed:
64 fix dimentions hover

elzadj committed:
65) map marker

elzadj committed:
66) Fix advances css textarea

kyrapieterse committed:
69) I guess, would be better to make black colour for text of custom link

kyrapieterse committed:
small fix

elzadj committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

elzadj committed:
36.29) Toolbar > alignment hover

elzadj committed:
Slider toolbar hover

elzadj committed:
68 accordion icon position buttons

elzadj committed:
70) video icon position dropdown

elzadj committed:
71 starter sites - hover and links

Olekrut commented:
> > 75. Need to add transition for icons transform
> 
> These are images, not svg.

@elzadj 
Why? we have these SVGs? weird

elzadj committed:
Fix auth and cloud for quick start

elzadj committed:
Fix broken sc editor animation

elzadj commented:
- [ ] First 'Quick Start' step is empty if we don't have any critical warnings to show
- [ ] Logging in during 'Quick start' reload 'Starter site' step and closes the model with starters 
- [ ] The menu in the imported header is really broken until you edit it (add an existing menu).

elzadj committed:
Fix php version needed

elzadj committed:
An attempts to fix SC import

elzadj committed:
Fix php and sql versions

elzadj committed:
Fix first  Quick Start' step being  empty if we don't have any critical warnings to show

elzadj committed:
Small hover fixes for starters and quick start

elzadj committed:
Fix starters modal closing after auth

elzadj committed:
Fix a php warning

elzadj committed:
fix sc import losing %

elzadj committed:
Fix sc buttons import

elzadj commented:
@Olekrut please test started sites that were added. Remove test started sites (draft them, I guess). 
Note: sc for Conference is weird, why is it named 'Maxi (Default) - Conference exported', and its structure is different?

Please check for https://github.com/maxi-blocks/maxi-blocks/pull/5880#issuecomment-2898519791

Olekrut reviewed (approved):
I ve found a few UI points, but will create new issue for them after
Amazing job!!!!!!!!!!!!

elzadj committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into onboarding-redesign

elzadj committed:
Add errors handling to QS completion

elzadj committed:
Fix php warning about strip_tags

elzadj committed:
Fix system staus report

elzadj committed:
Remove not-needed save_theme_settings function

elzadj committed:
More secured QS

elzadj committed:
Fix potential php warning

elzadj committed:
More secured status report

elzadj committed:
Fix translations issues

elzadj committed:
Add mariadb check for critical warnings to status report

elzadj committed:
Fix files names

elzadj committed:
Remove localhost from maxi go tempalte

elzadj committed:
Fix QS translation and escapes

elzadj committed:
Fix QS

elzadj committed:
Fix sslverify for sites and localhost

elzadj committed:
remove redundant files

elzadj committed:
Fix MIME bug with vars

elzadj committed:
Fix production detection

elzadj committed:
Fix ajax check

elzadj committed:
Fix QS first step redirection

elzadj committed:
Fix strip_tags php warning on php 8.2+ for quick start

elzadj committed:
Fix styles for Inter font

elzadj committed:
Better handling of auth cookie

elzadj committed:
Fix XSS problem in status report

elzadj committed:
Better auth for starters
